packages/gatsby-plugin-utils/src/polyfill-remote-file/graphql/gatsby-image-resolver.ts

```diff
@@ -210,13 +210,13 @@ export async function gatsbyImageResolver(
         srcSet: createSrcSetFromImages(images),
         sizes,
       }
-    } else {
```

Comment:
@wardpeet this if/else was causing the `auto` format to not push any sources and only have a fallback, failing the WP tests. This change fixes it but I don't really understand why you added an if/else here. Do you have context on why it was needed? Will this break something else unexpected?

Reply:
Alright, looks like it "broke" other tests. On further inspection looks like those tests had incorrect assertions. They were asserting that the only 2 formats were avif and webp, but as far as I know they also needed jpg (or png) since not all browsers support avif and webp. This was because `auto` format didn't work before this commit. So I believe removing this else is the right thing to do

Reply:
I have to re-look at the outcome.